contrib/mathtex-script-type/README.md

```diff
@@ -0,0 +1,16 @@
+# `math/tex` Custom Script Type Extension
+
+This is an extension to automatically display code inside `script` tags with `type=math/tex` using KaTeX.
+This script type is commonly used by MathJax, so this can be used to support compatibility with MathJax.
+
+### Usage
+
+This extension isn't part of KaTeX proper, so the script should be separately
+included in the page.
+This include should be loaded after all `script` blocks you want to render.
+
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/contrib/mathtex-script-type.min.js" integrity="sha384-o+v+EkJWQmZj7XwHBxehTGJKE18182WyyN2glZMTPw9g5XxjN1uwrquNuMX/NJiF"></script>
```

Comment:
Oops, indeed, `update-sri.js` already updates the version number.  It looks like it will replace any URL with any version number, including `...`, so we could still use that until a release happens.  Then the release will fix it to a working version number.